src/specs/KeyboardControllerViewNativeComponent.ts

```diff
@@ -16,9 +16,10 @@ export interface NativeProps extends ViewProps {
   statusBarTranslucent?: boolean;
   navigationBarTranslucent?: boolean;
   // callbacks
-  onKeyboardMove?: DirectEventHandler<KeyboardMoveEvent>;
   onKeyboardMoveStart?: DirectEventHandler<KeyboardMoveEvent>;
+  onKeyboardMove?: DirectEventHandler<KeyboardMoveEvent>;
   onKeyboardMoveEnd?: DirectEventHandler<KeyboardMoveEvent>;
+  onKeyboardMoveInteractive?: DirectEventHandler<KeyboardMoveEvent>;
```

Comment:
Read more about Partial<> in TS!

Reply:
I'm afraid specs/codegen may not support `Partial` type, so I would stick with an approach of duplication "?" symbols :)